substrate/client/authority-discovery/src/worker/schema/dht-v3.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package authority_discovery_v3;
```

Comment:
dq: Do we need to change a spec for this? Just to avoid silent changes for others (like lightclient).